### ISSUE TITLE
Figure out who the package maintainer should be

### ISSUE THREAD

**Issue by mjskay** [MUCollective/multiverse#27]
And give this person the "cre" tag.

(this is low priority but would need to be done before a CRAN submission)

**Comment by mjskay** [comment]
It seems like the obvious choice is @abhsarma (who already has the tag anyway) so i'd say we close this?

**Comment by abhsarma** [comment]
Sure, sounds good. I'll go ahead and close it then.
